packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button.stories.ts

```diff
@@ -0,0 +1,187 @@
+import { html, ref } from '@microsoft/fast-element';
+import type { HtmlRenderer, Meta, StoryObj } from '@storybook/html';
+import { withActions } from '@storybook/addon-actions/decorator';
+import { tableTag } from '../../../../../nimble-components/src/table';
+import { tableColumnTextTag } from '../../../../../nimble-components/src/table-column/text';
+import { tableColumnMenuButtonTag } from '../../../../../nimble-components/src/table-column/menu-button';
+import type { MenuButtonColumnToggleEventDetail } from '../../../../../nimble-components/src/table-column/menu-button/types';
+import { Menu, menuTag } from '../../../../../nimble-components/src/menu';
+import { menuItemTag } from '../../../../../nimble-components/src/menu-item';
+import {
+    SharedTableArgs,
+    sharedTableActions,
+    sharedTableArgTypes,
+    sharedTableArgs
+} from '../base/table-column-stories-utils';
+import {
+    apiCategory,
+    createUserSelectedThemeStory
+} from '../../../utilities/storybook';
+
+const simpleData = [
```

Comment:
What do you think about a couple changes to make it easier to experiment with interaction corner cases?
1. add more data so we can see what happens if an open menu is scrolled out of view
2. add an action menu so we can play with keyboard navigation and multiple popups
3. enable selection so we can see how it renders when the row is selected

I'm on the fence; these would more be for me as a tester and might complicate the user-facing documentation / code sample. Maybe I should just pull your branch and try those things myself, but I'm curious if you think others would find them useful.

Reply:
1. I've added a lot more data to the table (thanks Copilot for generating it all for me 😄)
2. I don't mind adding an action menu, but I wasn't sure how helpful that would be generally for a documentation point of view
3. Similar to the action menu. I don't mind adding selection to the table. I was just trying to be consistent with our other column type stories that don't do anything with selection.

Reply:
Wow, Copilot knows the names of Simpsons characters I'd never heard of!

The more I think about 2 and 3, the less I'm able to justify them for clients so I'm ok to leave them off in the name of consistency and simplicity.